code/__HELPERS/roundend.dm

```diff
@@ -218,12 +218,13 @@
 		speed_round = TRUE
 
 	for(var/client/C in GLOB.clients)
-		if(!C.credits)
-			C.RollCredits()
-		C.playtitlemusic(40)
-		if(speed_round)
-			C.give_award(/datum/award/achievement/misc/speed_round, C.mob)
-		HandleRandomHardcoreScore(C)
+		if(C)
```

Comment:
This is pointless, everything we do inside here uses null-coalescing accesses, and clients can delete themselves at any time (I've been told, anyway).